container-template/proxy/nginx.conf

```diff
@@ -27,7 +27,7 @@ http {
     # Fast Stable Diffusion + web UI + Comfy UI
     server {
         listen 3001;
-
+	client_max_body_size 1024M;
```

Comment:
Why do you want to set this to 1GB and not just 20MB which should be more than sufficient?
It would also be better to place this in the `http` section than `server ` section so all the other containers can also benefit from it.

Reply:
I want to be able to upload larger video files. to use with vid2vid type extensions.

I moved the setting to the server section.

I tested with and without going through the cloud flare proxy with this setting.
  - with the proxy, files up to 100MB seem to be allow, I assume that is due to the cloudflare setting.
  - without the proxy, and by directly exposing the tcp port during pod setup, I am able to upload files up to 1024MB